obs-studio-server/source/nodeobs_api.h

```diff
@@ -158,3 +158,18 @@ class OBS_API
 	static void SetProcessPriority(const char* priority);
 	static void destroyOBS_API(void);
 };
+
+class outdated_driver_error 
+{
+	static outdated_driver_error * inst;
+	std::string line_1 = ""; 
+	std::string line_2 = "";
+	int lookup_enabled = 0;
```

Comment:
this variable is meant to be a bool no?

Reply:
No, as it is used as a counter. 
For example if two threads requested to enable lookup for driver error. It will be disabled only after two request for disabling be received.

Reply:
I see, how can this happen though? All of this logic is in the main thread no?

Reply:
Yes. For this exact error we receive it immediately after obs_output_start() call. And as it only in our thread it cannot overlap. If it changed or we use it for other ffmpeg errors code will work without change. 
Complexity of this flag-counter is contained inside outdated_driver_error class and not making outside code more complicated.